File: richest-customer-wealth/richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        
        int maxWealth = Integer.MIN_VALUE;
        
        for (int[] a : accounts) {
            
            maxWealth = Math.max(maxWealth, Arrays.stream(a).sum());
        }
        
        return maxWealth;
    }
}

File: richest-customer-wealth/richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(const vector<vector<int>>& accounts) {
        
        int maxWealth = INT_MIN;
        
        for (const vector<int>& a : accounts) {
            
            maxWealth = max(maxWealth, accumulate(begin(a), end(a), 0));
        }
        
        return maxWealth;
    }
};